charts/kminion/templates/daemonset.yaml

```
@@ -47,6 +47,9 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8}}
       serviceAccountName: {{ include "kminion.serviceAccountName" . }}
+      {{- if .Values.priorityClassName }}
```

NIT: I'm more used to 
```suggestion
      {{- wiith .Values.priorityClassName }}
```

---

done :+1: